oak_marketplace_emotion_recognition/README.md

```diff
@@ -0,0 +1,42 @@
+This contains the code for \***\*. For more information - visit [\*\***]()
```

Comment:
For now, let's keep the following so we don't have broken links. We'll change that after publishing the post:

```
This contains the code for **Emotion Recognition using OAK and Modelplace**. For more information - visit [OpenCV.AI](https://www.opencv.ai/).
```